file-events/src/file-events/cpp/linux_fsnotifier.cpp

```diff
@@ -298,11 +298,10 @@ void Server::registerPath(const u16string& path) {
     if (watchRoots.find(watchDescriptor) != watchRoots.end()) {
         throw FileWatcherException("Already watching path", path);
     }
-    auto result = watchPoints.emplace(piecewise_construct,
+    watchPoints.emplace(piecewise_construct,
```

Comment:
I think we should add a comment here, why we don't re-use the return value of emplace.

Reply:
I don't think that's necessary. IIUC, it's quite non-typical to do so actually.